### PR TITLE
Removed map control from player statistics for performance reasons

### LINKED ISSUE #7685: PlayerStatistics update is a performance hog
Even just me without any other AI or player, the occasional ```PlayerStatistics``` update in skirmish takes ~23 ms, causing a short, but noticable lag spike. Appears to be related to render.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/9728820-playerstatistics-update-is-a-performance-hog?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by reaperrr**
Closes #7685.

### CODE REVIEW

File: mods/cnc/chrome/ingame-observerstats.yaml [hunk 1]
```diff
@@ -440,41 +440,36 @@ Background@INGAME_OBSERVERSTATS_BG:
 									Width: 160
 									Height: PARENT_BOTTOM
 									Font: Bold
-								Label@CONTROL:
```
Comment (obrakmann) [inline mods/cnc/chrome/ingame-observerstats.yaml]: You need to remove the `CONTROL_HEADER`, too

Comment (reaperrr) [inline mods/cnc/chrome/ingame-observerstats.yaml]: Yeah, I rushed this a little too much. Updated.

**Comment by Mailaender** [comment]
:+1: Tested all mods. :white_check_mark:

**Comment by obrakmann** [comment]
Looks good now. :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/c82271d09d2b3add7b616e56b4d19ea402775642%5E...c82271d09d2b3add7b616e56b4d19ea402775642)

**Pull request merged**